src/request.spec.js

```diff
@@ -105,6 +119,73 @@ describe('Request', function() {
     });
   });
 
+  describe('_attachInterceptors', function() {
+    let expectedRequestInterceptors;
+    let expectedResponseInterceptors;
+    let requestUse;
+    let responseUse;
+
+    beforeEach(function() {
+      /* eslint-disable no-new-func */
+      const requestInterceptors = times(
+        faker.random.number({ min: 0, max: 10 }),
+        () => {
+          return {
+            fulfilled: new Function(),
+            rejected: new Function()
+          };
+        }
+      );
+      const responseInterceptors = times(
+        faker.random.number({ min: 0, max: 10 }),
+        () => {
+          return {
+            fulfilled: new Function(),
+            rejected: new Function()
```

Comment:
Could you use `this.sandbox.stub()` here instead of `new Function()`?

Reply:
Yes, for sure. I had a brain fart trying to think of some method in faker. Somehow didn't make it to sinon.